integrations/astra/pyproject.toml

```diff
@@ -75,6 +77,7 @@ dependencies = [
   "black>=23.1.0",
   "mypy>=1.0.0",
   "ruff>=0.0.243",
+  "astrapy>=0.7.0",
```

Comment:
The jobs depends on the integration already, that depends on `astra`, no need for the extra dep here
```suggestion
```